src/Aeon/Emulator/Video/Rendering/GraphicsPresenterX.cs

```diff
@@ -0,0 +1,80 @@
+using System.Buffers.Binary;
+
+namespace Aeon.Emulator.Video.Rendering
+{
+    
+
+    /// <summary>
+    /// Renders 8-bit mode X graphics to a bitmap.
+    /// </summary>
+    public sealed class GraphicsPresenterX : Presenter
+    {
+        /// <summary>
+        /// Initializes a new instance of the GraphicsPresenterX class.
+        /// </summary>
+        /// <param name="dest">Pointer to destination bitmap.</param>
+        /// <param name="videoMode">VideoMode instance describing the video mode.</param>
+        public GraphicsPresenterX(VideoMode videoMode) : base(videoMode)
+        {
+        }
+
+        /// <summary>
+        /// Updates the bitmap to match the current state of the video RAM.
+        /// </summary>
+        protected override void DrawFrame(IntPtr destination)
+        {
+            int width = VideoMode.Width;
+            int height = VideoMode.Height;
+            var palette = VideoMode.Palette;
+            int startOffset = VideoMode.StartOffset;
+            int stride = VideoMode.Stride;
+            int lineCompare = VideoMode.LineCompare / 2;
+
+            unsafe
+            {
+                uint* destPtr = (uint*)destination.ToPointer();
+                uint* src = (uint*)VideoMode.VideoRam.ToPointer();
```

Comment:
All Presenters: Try to use Spans for source pointers. Only pointers for the destination buffer from Avalonia (since this can't be avoided).